okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/ThreadWorkQueue.java

```diff
@@ -40,6 +40,10 @@ boolean isEmpty() {
         return queuedRequests.isEmpty();
     }
 
+    int size() {
+        return queuedRequests.size();
```

Comment:
this gives you back the number of threads, not the number of queued requests

Reply:
Good spot - looks like we'd need to recurse through the map and sum up the size of all the per-thread queues.